README.adoc

```diff
@@ -53,170 +53,122 @@ include::{common-includes}/gitclone.adoc[]
-to report the state.
+The `@Health` annotation indicates that this particular bean implements the `HealthCheck` interface.
+By pairing this annotation with the `ApplicationScoped` context from the CDI API, the bean is discovered
+automatically when the `http://localhost:9080/health` endpoint receives a request.
 
-This class implements the `HealthCheck` interface from the MicroProfile Health API. The class also implements the `call()`
-method from the interface to report the state of the service by returning a `HealthCheckResponse` object. The `system`
-service self-checks. It reads the system property of the host to report a healthy state.
-If the `wlp.server.name` system property is `defaultServer`, the state is `UP`. Otherwise, the state is `DOWN`.
+The `call()` method is used to return the health status of a particular service. In this case, you are simply
+checking if the server name is `defaultServer` and returning `UP` if it is, and `DOWN` otherwise.
+The `HealthCheckResponse.named()` method is used to indicate what service the health check is done for.
+Overal, this is a very simple implementation of the `call()` method. In a real development environment,
+you would want to orchestrate much more meaningful health checks.
 
-Lastly, the `named()` method from the interface needs to indicate the service to report the health, thus,
-passing the `system` service name as an argument.
 
 // =================================================================================================
 // Adding a health check to the inventory service
 // =================================================================================================
 
-=== Adding a health check to the inventory service
+=== Adding a health check to the `inventory` service
 
-Create health check class for the `inventory` service in the
-`src/main/java/io/openliberty/guides/inventory/InventoryHealth.java` file:
+Implement the `HealthCheck` interface in the `src/main/java/io/openliberty/guides/inventory/InventoryHealth.java` file:
 
 [source, java, indent=0]
 ----
 include::finish/src/main/java/io/openliberty/guides/inventory/InventoryHealth.java[tags=InventoryHealth]
 ----
 
-This service reports a healthy state based on two factors:
+This time, you are checking whether or not the service is in maintenance or if it's down. For simplicity,
+a custom `io_openliberty_guides_inventory_inMaintenance` MicroProfile Config property defined in the
+`resources/CustomConfigSource.json` file is used to
+indicate whether the service is in maintenance or not. This file has already been created for you.
+To check that the service is down, simply make a HTTP GET request to itself and check the status
+returned by the response. If the status is not 200, then the service is not running. Based on these
+two factors, the `isHealthy()` method returns whether or not the `inventory` service is healthy.
```

Comment:
Not to itself but the `system` service?

Reply:
oh I see now, thats kind of confusing... We are reporting whether or not the `inventory` service is healthy by checking if the `system` service is running? I'll make a note of this somewhere so its clear the prior service depends on the latter one.

Reply:
Yes, that's my understanding.  Thank you.